inductiva/tasks/task.py

```diff
@@ -239,8 +239,9 @@ def __str__(self):
         table_str += f"\n{wall_time_table}"
         table_str += f"\nTime breakdown:\n{time_metrics_table}"
         table_str += f"\nData:\n{data_metrics_table}\n"
-        if self.cost:
-            table_str += f"\nCost ($US): {self.cost}\n"
+        if self.estimated_computation_cost:
+            table_str += ("\nEstimated computation cost ($US): "
```

Comment:
It should be `US$` and not `$US`:

```suggestion
            table_str += ("\nEstimated computation cost (US$): "
```